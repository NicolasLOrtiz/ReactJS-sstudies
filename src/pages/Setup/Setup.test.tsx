import { RecoilRoot } from "recoil";
import { render } from "@testing-library/react";
import React from "react";
import { Setup } from "./index";

const mockNavigate = jest.fn();
jest.mock("react-router-dom", () => ({
  useNavigate: () => mockNavigate,
}));
describe("Setup test suite", () => {
  it("should be able to render a setup page", () => {
    // Arrange
    const { container } = render(
      <RecoilRoot>
        <Setup />
      </RecoilRoot>
    );

    // Assert
    expect(container).toMatchSnapshot();
  });
});
