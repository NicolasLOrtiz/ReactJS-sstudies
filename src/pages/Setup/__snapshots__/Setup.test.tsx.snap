// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setup test suite should be able to render a setup page 1`] = `
<div>
  <form>
    <input
      placeholder="Nome completo"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
  </form>
  <ul />
  <footer
    role="contentinfo"
  >
    <button
      disabled=""
    >
      Sortear cliente
    </button>
  </footer>
</div>
`;
